src/app/src/components/EmbeddedMapConfigWrapper.jsx

```diff
@@ -108,7 +108,7 @@ function EmbeddedMapConfigWrapper({ user }) {
             } else {
                 createConfig();
             }
-        }, 500);
+        }, 1000);
```

Comment:
1000 ms is more practical to finish typing in the text input; 500 ms pause will cause a lag when typing a bit slower